nilearn/glm/first_level/design_matrix.py

```diff
@@ -510,7 +510,7 @@ def make_second_level_design_matrix(subjects_label, confounds=None):
                 design_matrix.loc[ridx, conf_name] = confounds_value
 
     # check design matrix is not singular
-    sys.float_info.epsilon
+    sys.float_info.epsilon  # noqa: B018
```

Comment:
I couldn't understand why we need this line here.

Reply:
this line is useless, right ?

Reply:
OK this has been there and untouched since nistats was merged into nilearn: no idea why this was put there.

will remove